CHANGELOG.md

```diff
@@ -1,6 +1,58 @@
 # Changelog
 All notable changes to this project will be documented in this file. This CHANGELOG roughly follows the guidelines from [www.keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [v0.10.2]
+### Added
+- Approximate font metrics only when metrics don't exist (#1898)
+- Add KaTeX version to stylesheet and troubleshooting guide (#1893)
+- Add symbol double square brackets (#1947)
+- Support double-square curly braces (#1953)
+
+ ### Changed
+- Upgrade minimum development Node version to v8 (#1861)
+- Disable @babel/env debug (#1874)
+- Add issue templates (#1862)
+- Added 'katex-element' (#1905)
+- Fix Users' logo and url (#1896)
+- Load fonts before running screenshotter (#1891)
+- Add Browserstack logo (#1879)
+- Added Android library (#1943)
+- Move custom colors used by Khan into macros.js (#1933)
+- Remove redundant symbols from #1947 (#1954)
+- Test for duplicate symbols/macros (#1955)
+- Include extensions mhchem & copy-tex in home-page (#1932)
+
+ ### Fixed
+- Fix \Rho (#1870)
+- Fix nested \dfrac (#1825)
+- Improve MathML accents (#1877)
+- Improve MathML for \overset, \stackrel, and \underset (#1886)
+- Fix \not (U+E020) RBearing (width) (#1878)
+- Fix ApplyFunction character (#1890)
+- Improve MathML for \limits (#1897)
+- Improve MathML for \hphantom and \vphantom (#1883)
+- Improve MathML for \coloneqq, \dblcolon, \eqcolon, and \eqqcolon (#1889)
+- Improve MathML for \brace (#1884)
+- Fix \middle spacing (#1906)
+- Get a tall \middle\vert from MathML (#1911)
+- Improve more coloneq (#1902)
+- Make \smallint small in \displaystyle (#1907)
+- Improve MathML for characters in Unicode private use area (#1908)
+- Improve MathML for extensible arrows (#1901)
+- Improve MathML for \rule (#1912)
+- Improve MathML for fractions (#1882)
+- Improve MathML for \tag (#1915)
+- Improve MathML for \colorbox and \fcolorbox (#1914)
+- Improve MathML for environments (#1910)
+- Improve MathML for \genfrac barline (#1925)
+- Support \textup and \textmd (#1921) 
+- Improve MathML for \not (#1923)
+- Improve MathML for \Bbbk (#1930)
+- Prevent inadvertent tall delims (#1948)
+
+ ### Removed
+- Disable \includegraphics until trust settings is merged (#1951) 
```

Comment:
Perhaps worth mentioning the PR that added `\includegraphics` too? This release simultaneously does both, but only the removal is mentioned in the changelog (unlike the PR description which mentions both).

Reply:
I'll modify this to say: `Re-added code for \includegraphics and disabled it until...`